### ISSUE TITLE
Keeping scripts in order

### ISSUE THREAD

**Issue by baboon-king** [zhazhazhu/ni#3]
I have same scripts in package.json

```json
  "scripts": {
    "prepare": "husky install",
    "format": "prettier --write .",
    "build": "zx ./scripts/build.mjs",
    "snapshot": "zx ./scripts/snapshot.mjs",
    "pretest": "run-s build snapshot",
    "test": "zx ./scripts/test.mjs",
    "prepublishOnly": "zx ./scripts/prepublish.mjs"
  },
```
Then run `nr` in command line
I got it. see image
![image](https://github.com/zhazhazhu/ni/assets/63645337/55e9493d-4f01-421a-bdca-c8e536a98e39)

**Comment by zhazhazhu** [comment]
This feature already exists, and I found a bug that caused it to not work. Thank you for your question

**Comment by zhazhazhu** [comment]
Will be fixed in the next version

**Comment by zhazhazhu** [comment]
Fixed in v0.0.10 version
